CONTRIBUTION_EXAMPLE.md

```diff
@@ -0,0 +1,32 @@
+# Example Contribution
```

Comment:
Please remove this. We have similar steps outlined in the CONTRIBUTING file. If you would like to propose changes there, please do it in a standalone PR.